packages/evm/src/provider/rpc.ts

```diff
@@ -102,6 +121,56 @@ export const ChainData: { [key: number]: RpcType } = {
     },
     blockExplorerUrls: ['https://bscscan.com'],
   },
+  [RpcId.BASE]: {
+    chainId: decToHex(8453),
+    rpcUrls: ['https://mainnet.base.org'],
+    chainName: 'Base Mainnet',
+    nativeCurrency: {
+      name: 'ETH',
+      symbol: 'ETH',
+      decimals: 18,
+    },
+    blockExplorerUrls: ['https://basescan.org'],
+  },
+  [RpcId.BASE_TESTNET]: {
+    chainId: decToHex(84532),
+    rpcUrls: ['https://sepolia.base.org'],
+    chainName: 'Base Sepolia (Testnet)',
+    nativeCurrency: {
+      name: 'ETH',
+      symbol: 'ETH',
+      decimals: 18,
+    },
+    blockExplorerUrls: ['	https://sepolia-explorer.base.org'],
+  },
+}
+
+export const ChainMetadata: { [key: number]: ChainMetadataType } = {
+  ...ChainData,
+  [RpcId.AVAX]: {
+    ...ChainData[RpcId.AVAX],
+    tokenAddress: '0xc0Fbc4967259786C743361a5885ef49380473dCF',
+    superTokenAddress: '0xc0Fbc4967259786C743361a5885ef49380473dCF',
+  },
+  [RpcId.AVAX_TESTNET]: {
+    ...ChainData[RpcId.AVAX_TESTNET],
+    tokenAddress: '0x1290248E01ED2F9f863A9752A8aAD396ef3a1B00',
+    superTokenAddress: '0x1290248E01ED2F9f863A9752A8aAD396ef3a1B00',
+  },
+  [RpcId.ETH_SEPOLIA]: {
+    ...ChainData[RpcId.ETH_SEPOLIA],
+    tokenAddress: '0xc4bf5cbdabe595361438f8c6a187bdc330539c60',
+    superTokenAddress: '0x22064a21fee226d8ffb8818e7627d5ff6d0fc33a',
+  },
+  [RpcId.BASE]: {
+    ...ChainData[RpcId.BASE],
+    tokenAddress: '0xc0Fbc4967259786C743361a5885ef49380473dCF',
+    superTokenAddress: '0xc0Fbc4967259786C743361a5885ef49380473dCF',
+  },
+}
```

Comment:
Added this `ChainMetadata` as a separate const because we are forwarding the `ChainData` to the provider and it was giving errors for the unsupported fields